molecule_vagrant/modules/vagrant.py

```diff
@@ -110,8 +110,9 @@
   provider_name:
     description:
       - Name of the Vagrant provider to use.
+      - If not set, let vagrant choose the provider and settings are done for virtualbox.
     required: False
-    default: virtualbox
+    default: None
```

Comment:
IMHO this looks very good, in order to sync with Vagrant default behavior.

Reply:
great. We're making progress :)